tests/vmtests/vmbackedtestcase.py

```diff
@@ -49,6 +49,9 @@ def set_up_storage(self):
             This will write the configuration to whatever disk images are
             defined in set_up_disks.
         """
+
+        udev.device_name_blacklist = [r'^zram']
```

Comment:
We'll to change this in #863

Reply:
Should we add this to the main list in `blivet.udev`?

Reply:
It is a block device and it can be used for other things than swap -- [documentation suggests](https://www.kernel.org/doc/Documentation/blockdev/zram.txt) using it for `/tmp` or `/var` so people might want to use blivet-gui to format it for example.